spark/src/test/scala/org/apache/hadoop/fs/versioned/S3TestSuite.scala

```diff
@@ -0,0 +1,53 @@
+package org.apache.hadoop.fs.versioned
+
+import com.amazonaws.auth.{AWSStaticCredentialsProvider, AnonymousAWSCredentials}
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.gu.tableversions.spark.SparkHiveSuite
+import io.findify.s3mock.S3Mock
+import org.scalatest.{BeforeAndAfterAll, Suite}
+
+trait S3TestSuite extends BeforeAndAfterAll {
```

Comment:
Do we still need this? I see it's mixed in to a test but doesn't seem used?

Reply:
Actually, I found the place where it's used. I don't think it adds much value over just using local files as the underlying filesystem?

Reply:
I removed it